components/financial_assistance/app/controllers/financial_assistance/incomes_controller.rb

```diff
@@ -139,12 +138,12 @@ def enable_bs4_layout
 
     def resolve_layout
       case action_name
-      when "index", "step", "new"
+      when "step", "new"
         "financial_assistance_nav"
-      when "other"
-        EnrollRegistry.feature_enabled?(:bs4_consumer_flow) ? "financial_assistance_progress" : "financial_assistance"
+      when "index", "other"
+        EnrollRegistry.feature_enabled?(:bs4_consumer_flow) ? "financial_assistance_progress" : "financial_assistance_nav"
       else
-        "financial_assistance"
+        "financial_assistance_nav"
```

Comment:
is this correct? which actions should be using this layout irrespective of the flag?

seems weird to change this when they were using `financial_assistance` before

Reply:
without this change, there is no nav for non-bootstrap layout